core/base/test/TStringTest.cxx

```diff
@@ -0,0 +1,15 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
```

Comment:
no need for this include

Reply:
Both of them?

Reply:
no just `gmock`, `gtest.h` brings in the `TEST` and `EXPECT_*` macros that we use